src/jti_background.js

```diff
@@ -61,6 +64,35 @@ function saveTemplates (templateJSON, callback, responseData = null) {
     });
 }
 
+function getDomains (callback) {
+    var domainArray = {};
+    // Get the default domains
+    var domainArray2 = $.map(DefaultDomainList, function (domain, index) {
+        domain.default = true;
```

Comment:
I'm not quite sure what you mean. You mean using the mapping function were `DefaultDomainList` is applied, so that it already holds the `domain.default = true`?

It's definitely possible, and I will discuss this with you. In my opinion I like having both the mappings together so that if someone wants to change one later they're in the same place (`getDomains` function).